views/choral.pug

```diff
@@ -98,9 +98,9 @@ block content
         if(childData.hasOwnProperty(data.device_id)) {
           var device_data = data.device_data;
           for( var key in device_data ) {
-            var data_value = device_data[key];
+            var data_value = device_data[key].toFixed(5);
```

Comment:
As I learned this doesn't actually fix it. 

If you have 3000.55555 that will still overflow.